docs/development-setup/deploying-a-runtime.md

```diff
@@ -352,7 +352,7 @@ production setting as they may result in node compromise.
 :::info
 
 When running a runtime node in a production setting, the
-`worker.p2p.addresses` and `worker.client.addresses` flags need to be configured
+`p2p.addresses` and `worker.client.addresses` flags need to be configured
```

Comment:
The `worker.client.addresses` does not need to be configured anymore, feel free to remove this part.

Reply:
The flag was removed.